onnxruntime/contrib_ops/cuda/quantization/qordered_ops/qordered_matmul.cc

```diff
@@ -98,9 +98,9 @@ Status QOrderedMatMul::PrePack(const Tensor& tensor, int input_idx, AllocatorPtr
       ORT_ENFORCE(tensor.Shape().NumDimensions() == 1 && tensor.Shape()[0] > 0, "bias must be 1d array!");
       const_bias_size_ = static_cast<int>(tensor.Shape().Size());
       const_bias_scaled_ = BufferUniquePtr(alloc->Alloc(const_bias_size_ * sizeof(float)), BufferDeleter(alloc));
-      CUBLAS_RETURN_IF_ERROR(cublasScopy(CublasHandle(), const_bias_size_, tensor.Data<float>(), 1, (float*)const_bias_scaled_.get(), 1));
+      CUBLAS_RETURN_IF_ERROR(cublasScopy(DefaultCublasHandle(), const_bias_size_, tensor.Data<float>(), 1, (float*)const_bias_scaled_.get(), 1));
       float rescale = static_cast<float>(1.0 / (double)const_scale_Y_);
-      CUBLAS_RETURN_IF_ERROR(cublasSscal(CublasHandle(), const_bias_size_, &rescale, (float*)const_bias_scaled_.get(), 1));
+      CUBLAS_RETURN_IF_ERROR(cublasSscal(DefaultCublasHandle(), const_bias_size_, &rescale, (float*)const_bias_scaled_.get(), 1));
```

Comment:
GetCublasHandle(context)? line 90, 92, 101, 103

Reply:
this during kernel initialization phase, context is not initialized yet, use DefaultCublasHandle is correct.